cernopendata/modules/fixtures/data/records/cms-tools-Phase2-datascience.json

```diff
@@ -29,8 +55,8 @@
       "formats": [
         "gz"
       ],
-      "number_files": 1,
-      "size": 0
+      "number_files": 472,
```

Comment:
This will be actually one file (the software tarball); I'll amend this while merging together with the JSON errors.